src/ripple/protocol/impl/BuildInfo.cpp

```diff
@@ -27,21 +27,17 @@ namespace ripple {
 
 namespace BuildInfo {
 
-char const* getRawVersionString ()
-{
-    static char const* const rawText =
+//------------------------------------------------------------------------------
+char const* const versionString =
```

Comment:
Oh, except that I guess your intention is for `versionString` to have extern linkage! :-)  I missed that on first read.  How do we grab `versionString` for the email report?